maml_rl/samplers/sampler.py

```diff
@@ -1,8 +1,8 @@
 import gym
 
-def make_env(env_name, env_kwargs={}, seed=None):
+def make_env(env_name, seed=None): #def make_env(env_name, env_kwargs={}, seed=None): - we don't have kwargs for tw-env
     def _make_env():
-        env = gym.make(env_name, **env_kwargs)
+        env = gym.make(env_name) #env = gym.make(env_name, **env_kwargs) -- we don't have kwargs for tw-env
```

Comment:
I don't think that should fail with tw-env. It'd be better to leave it there (i.e. reducing the changes).